### DISCUSSION TITLE
Multiple Price Graphs

### DISCUSSION THREAD

**Discussion by pwdel** [openpredictionmarkets/socialpredict#322] [Ideas]
We have an ongoing discussion related to Multiple Price graphs.

https://github.com/orgs/openpredictionmarkets/projects/3/views/1?pane=issue&itemId=79800929

If you would like to be added to this project specifically, please ask here. Alternatively please put your comments here on this thread.

**Comment by astrosnat** [comment]
Please add me. Also some thoughts:

- I personally find these confusing to read
- I like the way Metaculus does it where you can hover over a specific point in time and get a readout of what e.g. people thought on 29 February 2024
- I feel strongly about having an accessible, colorblind-friendly color palette: [one of many tools](https://venngage.com/tools/accessible-color-palette-generator)
